--- a
+++ b
@@ -1,6 +1,9 @@
 DROP INDEX trgm_idx2;
 \copy test_trgm3 from 'data/trgm2.data'
-ERROR: relation "test_trgm3" does not exist
+ERROR: failed to initiate COPY: failed to initiate COPY: Code: INTERNAL
+failed to initiate COPY: failed to create reserved connection for COPY: failed to initiate COPY FROM STDIN: ERROR: relation "test_trgm3" does not exist
+
+failed to receive READY response
 -- reduce noise
 set extra_float_digits = 0;
 select t,strict_word_similarity('Baykal',t) as sml from test_trgm2 where 'Baykal' <<% t order by sml desc, t;
