--- a
+++ b
@@ -157,57 +157,63 @@
 -- Foreign Data Wrapper and Foreign Server
 --
 CREATE FOREIGN DATA WRAPPER alt_fdw1;
+ERROR:  CREATE FOREIGN DATA WRAPPER is not supported through the connection pooler
 CREATE FOREIGN DATA WRAPPER alt_fdw2;
+ERROR:  CREATE FOREIGN DATA WRAPPER is not supported through the connection pooler
 CREATE SERVER alt_fserv1 FOREIGN DATA WRAPPER alt_fdw1;
+ERROR:  CREATE SERVER is not supported: creating foreign server connections is not permitted through the connection pooler
 CREATE SERVER alt_fserv2 FOREIGN DATA WRAPPER alt_fdw2;
+ERROR:  CREATE SERVER is not supported: creating foreign server connections is not permitted through the connection pooler
 ALTER FOREIGN DATA WRAPPER alt_fdw1 RENAME TO alt_fdw2;  -- failed (name conflict)
-ERROR:  foreign-data wrapper "alt_fdw2" already exists
+ERROR:  foreign-data wrapper "alt_fdw1" does not exist
 ALTER FOREIGN DATA WRAPPER alt_fdw1 RENAME TO alt_fdw3;  -- OK
+ERROR:  foreign-data wrapper "alt_fdw1" does not exist
 ALTER SERVER alt_fserv1 RENAME TO alt_fserv2;   -- failed (name conflict)
-ERROR:  server "alt_fserv2" already exists
+ERROR:  server "alt_fserv1" does not exist
 ALTER SERVER alt_fserv1 RENAME TO alt_fserv3;   -- OK
+ERROR:  server "alt_fserv1" does not exist
 SELECT fdwname FROM pg_foreign_data_wrapper WHERE fdwname like 'alt_fdw%';
- fdwname  
-----------
- alt_fdw2
- alt_fdw3
-(2 rows)
+ fdwname 
+---------
+(0 rows)
 
 SELECT srvname FROM pg_foreign_server WHERE srvname like 'alt_fserv%';
-  srvname   
-------------
- alt_fserv2
- alt_fserv3
-(2 rows)
+ srvname 
+---------
+(0 rows)
 
 --
 -- Procedural Language
 --
 CREATE LANGUAGE alt_lang1 HANDLER plpgsql_call_handler;
+ERROR:  CREATE LANGUAGE is not supported: installing procedural languages is not permitted through the connection pooler
 CREATE LANGUAGE alt_lang2 HANDLER plpgsql_call_handler;
+ERROR:  CREATE LANGUAGE is not supported: installing procedural languages is not permitted through the connection pooler
 ALTER LANGUAGE alt_lang1 OWNER TO regress_alter_generic_user1;  -- OK
+ERROR:  language "alt_lang1" does not exist
 ALTER LANGUAGE alt_lang2 OWNER TO regress_alter_generic_user2;  -- OK
+ERROR:  language "alt_lang2" does not exist
 SET SESSION AUTHORIZATION regress_alter_generic_user1;
 ALTER LANGUAGE alt_lang1 RENAME TO alt_lang2;   -- failed (name conflict)
-ERROR:  language "alt_lang2" already exists
+ERROR:  language "alt_lang1" does not exist
 ALTER LANGUAGE alt_lang2 RENAME TO alt_lang3;   -- failed (not owner)
-ERROR:  must be owner of language alt_lang2
+ERROR:  language "alt_lang2" does not exist
 ALTER LANGUAGE alt_lang1 RENAME TO alt_lang3;   -- OK
+ERROR:  language "alt_lang1" does not exist
 ALTER LANGUAGE alt_lang2 OWNER TO regress_alter_generic_user3;  -- failed (not owner)
-ERROR:  must be owner of language alt_lang2
+ERROR:  language "alt_lang2" does not exist
 ALTER LANGUAGE alt_lang3 OWNER TO regress_alter_generic_user2;  -- failed (no role membership)
-ERROR:  must be able to SET ROLE "regress_alter_generic_user2"
+ERROR:  language "alt_lang3" does not exist
 ALTER LANGUAGE alt_lang3 OWNER TO regress_alter_generic_user3;  -- OK
+ERROR:  language "alt_lang3" does not exist
 RESET SESSION AUTHORIZATION;
 SELECT lanname, a.rolname
   FROM pg_language l, pg_authid a
   WHERE l.lanowner = a.oid AND l.lanname like 'alt_lang%'
   ORDER BY lanname;
-  lanname  |           rolname           
------------+-----------------------------
- alt_lang2 | regress_alter_generic_user2
- alt_lang3 | regress_alter_generic_user3
-(2 rows)
+ lanname | rolname 
+---------+---------
+(0 rows)
 
 --
 -- Operator
@@ -392,29 +398,44 @@
 ROLLBACK;
 -- Should fail. Only two arguments required for ALTER OPERATOR FAMILY ... DROP OPERATOR
 CREATE OPERATOR FAMILY alt_opf7 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf7 USING btree ADD OPERATOR 1 < (int4, int2);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf7 USING btree DROP OPERATOR 1 (int4, int2, int8);
-ERROR:  one or two argument types must be specified
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf7 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should work. During ALTER OPERATOR FAMILY ... DROP OPERATOR
 -- when left type is the same as right type, a DROP with only one argument type should work
 CREATE OPERATOR FAMILY alt_opf8 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf8 USING btree ADD OPERATOR 1 < (int4, int4);
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf8 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should work. Textbook case of ALTER OPERATOR FAMILY ... ADD OPERATOR with FOR ORDER BY
 CREATE OPERATOR FAMILY alt_opf9 USING gist;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf9 USING gist ADD OPERATOR 1 < (int4, int4) FOR ORDER BY float_ops;
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf9 USING gist;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should fail. Ensure correct ordering methods in ALTER OPERATOR FAMILY ... ADD OPERATOR .. FOR ORDER BY
 CREATE OPERATOR FAMILY alt_opf10 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf10 USING btree ADD OPERATOR 1 < (int4, int4) FOR ORDER BY float_ops;
-ERROR:  access method "btree" does not support ordering operators
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf10 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should work. Textbook case of ALTER OPERATOR FAMILY ... ADD OPERATOR with FOR ORDER BY
 CREATE OPERATOR FAMILY alt_opf11 USING gist;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf11 USING gist ADD OPERATOR 1 < (int4, int4) FOR ORDER BY float_ops;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf11 USING gist DROP OPERATOR 1 (int4, int4);
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf11 USING gist;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should fail. btree comparison functions should return INTEGER in ALTER OPERATOR FAMILY ... ADD FUNCTION
 BEGIN TRANSACTION;
 CREATE OPERATOR FAMILY alt_opf12 USING btree;
@@ -454,16 +475,20 @@
 -- Should fail. In gist throw an error when giving different data types for function argument
 -- without defining left / right type in ALTER OPERATOR FAMILY ... ADD FUNCTION
 CREATE OPERATOR FAMILY alt_opf16 USING gist;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf16 USING gist ADD FUNCTION 1 btint42cmp(int4, int2);
-ERROR:  associated data types must be specified for index support function
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf16 USING gist;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should fail. duplicate operator number / function number in ALTER OPERATOR FAMILY ... ADD FUNCTION
 CREATE OPERATOR FAMILY alt_opf17 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf17 USING btree ADD OPERATOR 1 < (int4, int4), OPERATOR 1 < (int4, int4); -- operator # appears twice in same statement
-ERROR:  operator number 1 for (integer,integer) appears more than once
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf17 USING btree ADD OPERATOR 1 < (int4, int4); -- operator 1 requested first-time
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf17 USING btree ADD OPERATOR 1 < (int4, int4); -- operator 1 requested again in separate statement
-ERROR:  operator 1(integer,integer) already exists in operator family "alt_opf17"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf17 USING btree ADD
   OPERATOR 1 < (int4, int2) ,
   OPERATOR 2 <= (int4, int2) ,
@@ -472,7 +497,7 @@
   OPERATOR 5 > (int4, int2) ,
   FUNCTION 1 btint42cmp(int4, int2) ,
   FUNCTION 1 btint42cmp(int4, int2);    -- procedure 1 appears twice in same statement
-ERROR:  function number 1 for (integer,smallint) appears more than once
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf17 USING btree ADD
   OPERATOR 1 < (int4, int2) ,
   OPERATOR 2 <= (int4, int2) ,
@@ -480,6 +505,7 @@
   OPERATOR 4 >= (int4, int2) ,
   OPERATOR 5 > (int4, int2) ,
   FUNCTION 1 btint42cmp(int4, int2);    -- procedure 1 appears first time
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf17 USING btree ADD
   OPERATOR 1 < (int4, int2) ,
   OPERATOR 2 <= (int4, int2) ,
@@ -487,13 +513,15 @@
   OPERATOR 4 >= (int4, int2) ,
   OPERATOR 5 > (int4, int2) ,
   FUNCTION 1 btint42cmp(int4, int2);    -- procedure 1 requested again in separate statement
-ERROR:  operator 1(integer,smallint) already exists in operator family "alt_opf17"
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf17 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should fail. Ensure that DROP requests for missing OPERATOR / FUNCTIONS
 -- return appropriate message in ALTER OPERATOR FAMILY ... DROP OPERATOR / FUNCTION
 CREATE OPERATOR FAMILY alt_opf18 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf18 USING btree DROP OPERATOR 1 (int4, int4);
-ERROR:  operator 1(integer,integer) does not exist in operator family "alt_opf18"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf18 USING btree ADD
   OPERATOR 1 < (int4, int2) ,
   OPERATOR 2 <= (int4, int2) ,
@@ -501,255 +529,231 @@
   OPERATOR 4 >= (int4, int2) ,
   OPERATOR 5 > (int4, int2) ,
   FUNCTION 1 btint42cmp(int4, int2);
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should fail. Not allowed to have cross-type equalimage function.
 ALTER OPERATOR FAMILY alt_opf18 USING btree
   ADD FUNCTION 4 (int4, int2) btequalimage(oid);
-ERROR:  btree equal image functions must not be cross-type
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf18 USING btree DROP FUNCTION 2 (int4, int4);
-ERROR:  function 2(integer,integer) does not exist in operator family "alt_opf18"
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf18 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- Should fail. Invalid opclass options function (#5) specifications.
 CREATE OPERATOR FAMILY alt_opf19 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf19 USING btree ADD FUNCTION 5 test_opclass_options_func(internal, text[], bool);
-ERROR:  function test_opclass_options_func(internal, text[], boolean) does not exist
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf19 USING btree ADD FUNCTION 5 (int4) btint42cmp(int4, int2);
-ERROR:  invalid operator class options parsing function
-HINT:  Valid signature of operator class options parsing function is (internal) RETURNS void.
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf19 USING btree ADD FUNCTION 5 (int4, int2) btint42cmp(int4, int2);
-ERROR:  left and right associated data types for operator class options parsing functions must match
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf19 USING btree ADD FUNCTION 5 (int4) test_opclass_options_func(internal); -- Ok
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER OPERATOR FAMILY alt_opf19 USING btree DROP FUNCTION 5 (int4, int4);
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP OPERATOR FAMILY alt_opf19 USING btree;
+ERROR:  role "regress_alter_generic_user6" does not exist
 --
 -- Statistics
 --
 SET SESSION AUTHORIZATION regress_alter_generic_user1;
 CREATE TABLE alt_regress_1 (a INTEGER, b INTEGER);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE STATISTICS alt_stat1 ON a, b FROM alt_regress_1;
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE STATISTICS alt_stat2 ON a, b FROM alt_regress_1;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat1 RENAME TO alt_stat2;   -- failed (name conflict)
-ERROR:  statistics object "alt_stat2" already exists in schema "alt_nsp1"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat1 RENAME TO alt_stat3;   -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat2 OWNER TO regress_alter_generic_user2;  -- failed (no role membership)
-ERROR:  must be able to SET ROLE "regress_alter_generic_user2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat2 OWNER TO regress_alter_generic_user3;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat2 SET SCHEMA alt_nsp2;    -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 SET SESSION AUTHORIZATION regress_alter_generic_user2;
 CREATE TABLE alt_regress_2 (a INTEGER, b INTEGER);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE STATISTICS alt_stat1 ON a, b FROM alt_regress_2;
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE STATISTICS alt_stat2 ON a, b FROM alt_regress_2;
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat3 RENAME TO alt_stat4;    -- failed (not owner)
-ERROR:  must be owner of statistics object alt_stat3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat1 RENAME TO alt_stat4;    -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat3 OWNER TO regress_alter_generic_user2; -- failed (not owner)
-ERROR:  must be owner of statistics object alt_stat3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat2 OWNER TO regress_alter_generic_user3; -- failed (no role membership)
-ERROR:  must be able to SET ROLE "regress_alter_generic_user3"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat3 SET SCHEMA alt_nsp2;		-- failed (not owner)
-ERROR:  must be owner of statistics object alt_stat3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER STATISTICS alt_stat2 SET SCHEMA alt_nsp2;		-- failed (name conflict)
-ERROR:  statistics object "alt_stat2" already exists in schema "alt_nsp2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 RESET SESSION AUTHORIZATION;
 SELECT nspname, stxname, rolname
   FROM pg_statistic_ext s, pg_namespace n, pg_authid a
  WHERE s.stxnamespace = n.oid AND s.stxowner = a.oid
    AND n.nspname in ('alt_nsp1', 'alt_nsp2')
  ORDER BY nspname, stxname;
- nspname  |  stxname  |           rolname           
-----------+-----------+-----------------------------
- alt_nsp1 | alt_stat2 | regress_alter_generic_user2
- alt_nsp1 | alt_stat3 | regress_alter_generic_user1
- alt_nsp1 | alt_stat4 | regress_alter_generic_user2
- alt_nsp2 | alt_stat2 | regress_alter_generic_user3
-(4 rows)
-
+ERROR:  role "regress_alter_generic_user6" does not exist
 --
 -- Text Search Dictionary
 --
 SET SESSION AUTHORIZATION regress_alter_generic_user1;
 CREATE TEXT SEARCH DICTIONARY alt_ts_dict1 (template=simple);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH DICTIONARY alt_ts_dict2 (template=simple);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict1 RENAME TO alt_ts_dict2;  -- failed (name conflict)
-ERROR:  text search dictionary "alt_ts_dict2" already exists in schema "alt_nsp1"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict1 RENAME TO alt_ts_dict3;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict2 OWNER TO regress_alter_generic_user2;  -- failed (no role membership)
-ERROR:  must be able to SET ROLE "regress_alter_generic_user2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict2 OWNER TO regress_alter_generic_user3;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict2 SET SCHEMA alt_nsp2;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 SET SESSION AUTHORIZATION regress_alter_generic_user2;
 CREATE TEXT SEARCH DICTIONARY alt_ts_dict1 (template=simple);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH DICTIONARY alt_ts_dict2 (template=simple);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict3 RENAME TO alt_ts_dict4;  -- failed (not owner)
-ERROR:  must be owner of text search dictionary alt_ts_dict3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict1 RENAME TO alt_ts_dict4;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict3 OWNER TO regress_alter_generic_user2;  -- failed (not owner)
-ERROR:  must be owner of text search dictionary alt_ts_dict3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict2 OWNER TO regress_alter_generic_user3;  -- failed (no role membership)
-ERROR:  must be able to SET ROLE "regress_alter_generic_user3"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict3 SET SCHEMA alt_nsp2;  -- failed (not owner)
-ERROR:  must be owner of text search dictionary alt_ts_dict3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH DICTIONARY alt_ts_dict2 SET SCHEMA alt_nsp2;  -- failed (name conflict)
-ERROR:  text search dictionary "alt_ts_dict2" already exists in schema "alt_nsp2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 RESET SESSION AUTHORIZATION;
 SELECT nspname, dictname, rolname
   FROM pg_ts_dict t, pg_namespace n, pg_authid a
   WHERE t.dictnamespace = n.oid AND t.dictowner = a.oid
     AND n.nspname in ('alt_nsp1', 'alt_nsp2')
   ORDER BY nspname, dictname;
- nspname  |   dictname   |           rolname           
-----------+--------------+-----------------------------
- alt_nsp1 | alt_ts_dict2 | regress_alter_generic_user2
- alt_nsp1 | alt_ts_dict3 | regress_alter_generic_user1
- alt_nsp1 | alt_ts_dict4 | regress_alter_generic_user2
- alt_nsp2 | alt_ts_dict2 | regress_alter_generic_user3
-(4 rows)
-
+ERROR:  role "regress_alter_generic_user6" does not exist
 --
 -- Text Search Configuration
 --
 SET SESSION AUTHORIZATION regress_alter_generic_user1;
 CREATE TEXT SEARCH CONFIGURATION alt_ts_conf1 (copy=english);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH CONFIGURATION alt_ts_conf2 (copy=english);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf1 RENAME TO alt_ts_conf2;  -- failed (name conflict)
-ERROR:  text search configuration "alt_ts_conf2" already exists in schema "alt_nsp1"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf1 RENAME TO alt_ts_conf3;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf2 OWNER TO regress_alter_generic_user2;  -- failed (no role membership)
-ERROR:  must be able to SET ROLE "regress_alter_generic_user2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf2 OWNER TO regress_alter_generic_user3;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf2 SET SCHEMA alt_nsp2;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 SET SESSION AUTHORIZATION regress_alter_generic_user2;
 CREATE TEXT SEARCH CONFIGURATION alt_ts_conf1 (copy=english);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH CONFIGURATION alt_ts_conf2 (copy=english);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf3 RENAME TO alt_ts_conf4;  -- failed (not owner)
-ERROR:  must be owner of text search configuration alt_ts_conf3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf1 RENAME TO alt_ts_conf4;  -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf3 OWNER TO regress_alter_generic_user2;  -- failed (not owner)
-ERROR:  must be owner of text search configuration alt_ts_conf3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf2 OWNER TO regress_alter_generic_user3;  -- failed (no role membership)
-ERROR:  must be able to SET ROLE "regress_alter_generic_user3"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf3 SET SCHEMA alt_nsp2;  -- failed (not owner)
-ERROR:  must be owner of text search configuration alt_ts_conf3
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH CONFIGURATION alt_ts_conf2 SET SCHEMA alt_nsp2;  -- failed (name conflict)
-ERROR:  text search configuration "alt_ts_conf2" already exists in schema "alt_nsp2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 RESET SESSION AUTHORIZATION;
 SELECT nspname, cfgname, rolname
   FROM pg_ts_config t, pg_namespace n, pg_authid a
   WHERE t.cfgnamespace = n.oid AND t.cfgowner = a.oid
     AND n.nspname in ('alt_nsp1', 'alt_nsp2')
   ORDER BY nspname, cfgname;
- nspname  |   cfgname    |           rolname           
-----------+--------------+-----------------------------
- alt_nsp1 | alt_ts_conf2 | regress_alter_generic_user2
- alt_nsp1 | alt_ts_conf3 | regress_alter_generic_user1
- alt_nsp1 | alt_ts_conf4 | regress_alter_generic_user2
- alt_nsp2 | alt_ts_conf2 | regress_alter_generic_user3
-(4 rows)
-
+ERROR:  role "regress_alter_generic_user6" does not exist
 --
 -- Text Search Template
 --
 CREATE TEXT SEARCH TEMPLATE alt_ts_temp1 (lexize=dsimple_lexize);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH TEMPLATE alt_ts_temp2 (lexize=dsimple_lexize);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH TEMPLATE alt_ts_temp1 RENAME TO alt_ts_temp2; -- failed (name conflict)
-ERROR:  text search template "alt_ts_temp2" already exists in schema "alt_nsp1"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH TEMPLATE alt_ts_temp1 RENAME TO alt_ts_temp3; -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH TEMPLATE alt_ts_temp2 SET SCHEMA alt_nsp2;    -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH TEMPLATE alt_ts_temp2 (lexize=dsimple_lexize);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH TEMPLATE alt_ts_temp2 SET SCHEMA alt_nsp2;    -- failed (name conflict)
-ERROR:  text search template "alt_ts_temp2" already exists in schema "alt_nsp2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- invalid: non-lowercase quoted identifiers
 CREATE TEXT SEARCH TEMPLATE tstemp_case ("Init" = init_function);
-ERROR:  text search template parameter "Init" not recognized
+ERROR:  role "regress_alter_generic_user6" does not exist
 SELECT nspname, tmplname
   FROM pg_ts_template t, pg_namespace n
   WHERE t.tmplnamespace = n.oid AND nspname like 'alt_nsp%'
   ORDER BY nspname, tmplname;
- nspname  |   tmplname   
-----------+--------------
- alt_nsp1 | alt_ts_temp2
- alt_nsp1 | alt_ts_temp3
- alt_nsp2 | alt_ts_temp2
-(3 rows)
-
+ERROR:  role "regress_alter_generic_user6" does not exist
 --
 -- Text Search Parser
 --
 CREATE TEXT SEARCH PARSER alt_ts_prs1
     (start = prsd_start, gettoken = prsd_nexttoken, end = prsd_end, lextypes = prsd_lextype);
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH PARSER alt_ts_prs2
     (start = prsd_start, gettoken = prsd_nexttoken, end = prsd_end, lextypes = prsd_lextype);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH PARSER alt_ts_prs1 RENAME TO alt_ts_prs2; -- failed (name conflict)
-ERROR:  text search parser "alt_ts_prs2" already exists in schema "alt_nsp1"
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH PARSER alt_ts_prs1 RENAME TO alt_ts_prs3; -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH PARSER alt_ts_prs2 SET SCHEMA alt_nsp2;   -- OK
+ERROR:  role "regress_alter_generic_user6" does not exist
 CREATE TEXT SEARCH PARSER alt_ts_prs2
     (start = prsd_start, gettoken = prsd_nexttoken, end = prsd_end, lextypes = prsd_lextype);
+ERROR:  role "regress_alter_generic_user6" does not exist
 ALTER TEXT SEARCH PARSER alt_ts_prs2 SET SCHEMA alt_nsp2;   -- failed (name conflict)
-ERROR:  text search parser "alt_ts_prs2" already exists in schema "alt_nsp2"
+ERROR:  role "regress_alter_generic_user6" does not exist
 -- invalid: non-lowercase quoted identifiers
 CREATE TEXT SEARCH PARSER tspars_case ("Start" = start_function);
-ERROR:  text search parser parameter "Start" not recognized
+ERROR:  role "regress_alter_generic_user6" does not exist
 SELECT nspname, prsname
   FROM pg_ts_parser t, pg_namespace n
   WHERE t.prsnamespace = n.oid AND nspname like 'alt_nsp%'
   ORDER BY nspname, prsname;
- nspname  |   prsname   
-----------+-------------
- alt_nsp1 | alt_ts_prs2
- alt_nsp1 | alt_ts_prs3
- alt_nsp2 | alt_ts_prs2
-(3 rows)
-
+ERROR:  role "regress_alter_generic_user6" does not exist
 ---
 --- Cleanup resources
 ---
 DROP FOREIGN DATA WRAPPER alt_fdw2 CASCADE;
-NOTICE:  drop cascades to server alt_fserv2
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP FOREIGN DATA WRAPPER alt_fdw3 CASCADE;
-NOTICE:  drop cascades to server alt_fserv3
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP LANGUAGE alt_lang2 CASCADE;
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP LANGUAGE alt_lang3 CASCADE;
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP SCHEMA alt_nsp1 CASCADE;
-NOTICE:  drop cascades to 28 other objects
-DETAIL:  drop cascades to function alt_func3(integer)
-drop cascades to function alt_agg3(integer)
-drop cascades to function alt_func4(integer)
-drop cascades to function alt_func2(integer)
-drop cascades to function alt_agg4(integer)
-drop cascades to function alt_agg2(integer)
-drop cascades to conversion alt_conv3
-drop cascades to conversion alt_conv4
-drop cascades to conversion alt_conv2
-drop cascades to operator @+@(integer,integer)
-drop cascades to operator @-@(integer,integer)
-drop cascades to operator family alt_opf3 for access method hash
-drop cascades to operator family alt_opc1 for access method hash
-drop cascades to operator family alt_opc2 for access method hash
-drop cascades to operator family alt_opf4 for access method hash
-drop cascades to operator family alt_opf2 for access method hash
-drop cascades to table alt_regress_1
-drop cascades to table alt_regress_2
-drop cascades to text search dictionary alt_ts_dict3
-drop cascades to text search dictionary alt_ts_dict4
-drop cascades to text search dictionary alt_ts_dict2
-drop cascades to text search configuration alt_ts_conf3
-drop cascades to text search configuration alt_ts_conf4
-drop cascades to text search configuration alt_ts_conf2
-drop cascades to text search template alt_ts_temp3
-drop cascades to text search template alt_ts_temp2
-drop cascades to text search parser alt_ts_prs3
-drop cascades to text search parser alt_ts_prs2
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP SCHEMA alt_nsp2 CASCADE;
-NOTICE:  drop cascades to 9 other objects
-DETAIL:  drop cascades to function alt_nsp2.alt_func2(integer)
-drop cascades to function alt_nsp2.alt_agg2(integer)
-drop cascades to conversion alt_nsp2.alt_conv2
-drop cascades to operator alt_nsp2.@-@(integer,integer)
-drop cascades to operator family alt_nsp2.alt_opf2 for access method hash
-drop cascades to text search dictionary alt_nsp2.alt_ts_dict2
-drop cascades to text search configuration alt_nsp2.alt_ts_conf2
-drop cascades to text search template alt_nsp2.alt_ts_temp2
-drop cascades to text search parser alt_nsp2.alt_ts_prs2
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP USER regress_alter_generic_user1;
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP USER regress_alter_generic_user2;
+ERROR:  role "regress_alter_generic_user6" does not exist
 DROP USER regress_alter_generic_user3;
+ERROR:  role "regress_alter_generic_user6" does not exist
