--- a
+++ b
@@ -48,16 +48,12 @@
 -- syntax errors
 -- test for extraneous comma
 CREATE TABLE error_tbl (i int DEFAULT (100, ));
-ERROR: syntax error at or near ")"
-LINE 1: CREATE TABLE error_tbl (i int DEFAULT (100, ));
-^
+ERROR: parse error at position 45: syntax error
 -- this will fail because gram.y uses b_expr not a_expr for defaults,
 -- to avoid a shift/reduce conflict that arises from NOT NULL being
 -- part of the column definition syntax:
 CREATE TABLE error_tbl (b1 bool DEFAULT 1 IN (1, 2));
-ERROR: syntax error at or near "IN"
-LINE 1: CREATE TABLE error_tbl (b1 bool DEFAULT 1 IN (1, 2));
-^
+ERROR: parse error at position 44: syntax error
 -- this should work, however:
 CREATE TABLE error_tbl (b1 bool DEFAULT (1 IN (1, 2)));
 DROP TABLE error_tbl;
