--- a
+++ b
@@ -702,9 +702,7 @@
 ERROR: column "a" specified more than once
 CREATE TRIGGER error_ins_a BEFORE INSERT OF a ON main_table
 FOR EACH ROW EXECUTE PROCEDURE trigger_func('error_ins_a');
-ERROR: syntax error at or near "OF"
-LINE 1: CREATE TRIGGER error_ins_a BEFORE INSERT OF a ON main_table
-^
+ERROR: parse error at position 43: syntax error
 CREATE TRIGGER error_ins_when BEFORE INSERT OR UPDATE ON main_table
 FOR EACH ROW WHEN (OLD.a <> NEW.a)
 EXECUTE PROCEDURE trigger_func('error_ins_old');
@@ -3738,9 +3736,7 @@
 (4 rows)
 
 alter trigger a on only grandparent rename to b; -- ONLY not supported
-ERROR: syntax error at or near "only"
-LINE 1: alter trigger a on only grandparent rename to b;
-^
+ERROR: parse error at position 23: syntax error
 alter trigger b on middle rename to c; -- can't rename trigger on partition
 ERROR: cannot rename trigger "b" on table "middle"
 HINT: Rename the trigger on the partitioned table "grandparent" instead.
